python/src/prices.py

```diff
@@ -3,32 +3,43 @@
 from flask import Flask
 from flask import request
 from datetime import datetime
-from db import create_lift_pass_db_connection
+import pymysql.cursors
+
+# from db import create_lift_pass_db_connection
 
 app = Flask("lift-pass-pricing")
 
-connection_options = {"host": 'localhost', "user": 'root', "database": 'lift_pass', "password": 'mysql'}
-connection = create_lift_pass_db_connection(connection_options)
+connection_options = {"host": 'localhost', "user": 'root',
+                      "database": 'lift_pass', "password": 'mysql'}
+# connection = create_lift_pass_db_connection(connection_options)
+
+connection = pymysql.connect(host='localhost',
+                             user='root',
+                             password='mysql',
+                             database='lift_pass',
+                             cursorclass=pymysql.cursors.DictCursor)
```

Comment:
Switch this for a  pymysql.cursors.Cursor - it works much more similarly to how the odbc cursor